platform/base/core/src/main/resources/SLING-INF/nodetypes/pagecontent.cnd

```diff
@@ -26,4 +26,4 @@
 
 [per:PageContent] > nt:unstructured, sling:Resource, sling:VanityPath, per:Replication
   orderable
-
+  - siteLanguage (string) = "de" autocreated
```

Comment:
it may be better to set the default to en

Reply:
will change it, forgot to set it to english